src/openforms/js/components/admin/form_design/registrations/zgw/fields/ZGWAPIGroup.js

```diff
@@ -44,13 +44,16 @@ const ZGWAPIGroup = ({apiGroupChoices, onChangeCheck}) => {
         }
         noManageChildProps
       >
-        <ReactSelect
+        <SelectWithoutFormik
```

Comment:
Are these changes intentional? I'm not sure if we want to remove formik for this field

Reply:
Yes this is intentional. The ReactSelect should set the value after the confirmation is accepted. The `SelectWithFormik` (the component that is exported as the default, aka `ReactSelect`) doesn't allow custom onChange handling, seeing how Formik should control this component.

So another solution for this would be to add a `onChange` handler to the `SelectWithFormik`, but that would (in my opinion) kinda defeat the purpose of the component..

This field still uses Formik, but the value/change handling is now done by the component, instead of the Select

Reply:
Yeah, its still the case that the `SelectWithFormik` (at the moment) doesn't allow custom onChange handling. This is needed, to trigger the confirmation modal onChange, and to save the data to formik after the confirmation is accepted.

Reply:
Nevermind... I just noticed (with a comment from Sergei) that you can overwrite the props of the `SelectWithFormik`..
So yeah, this isn't needed and i'll revert this